Nero/src/NeroMet.cpp

```diff
@@ -75,10 +75,11 @@ int NeroMet::analyze(const edm::Event& iEvent){
         *metNoHF = TLorentzVector( metnohf.px(),metnohf.py(),metnohf.pz(),metnohf.energy());
         sumEtRawNoHF = metnohf.uncorSumEt();
 
-        const pat::MET &puppi = handle_puppi->front(); 
+        auto &puppi = handle_puppi->front(); 
         *metPuppi =  TLorentzVector( puppi.px(), puppi.py(),puppi.pz(),puppi.energy() );
-        sumEtRawPuppi = puppi.uncorSumEt();
+        sumEtRawPuppi = puppi.sumEt();
```

Comment:
You are right. The sumEt correction is added in [1]. The raw sum Et was added by somebody else, so I guess they must need it. I'll fix it.

[1] https://github.com/cms-sw/cmssw/blob/CMSSW_7_6_X/JetMETCorrections/Type1MET/src/AddCorrectionsToGenericMET.cc#L45